src/osd/PeeringState.h

```diff
@@ -1765,6 +1765,17 @@ class PeeringState : public MissingLoc::MappingInfo {
     std::optional<eversion_t> trim_to,
     std::optional<eversion_t> roll_forward_to);
 
+  void append_log_with_trim_to_updated(
```

Comment:
if we already expose both `update_trim_to()` and `append_log()` as public methods, why bother adding yet another helper for wrapping up them? i don't see it offers extra values aside from composing these two methods.

Reply:
Oh, the "append_log" method accepts as parameters "pg_trim_to" and "min_last_complete_ondisk" which are members of PeeringState, if we use append_log here, the calling would be like "pg.append_log(...,pg.get_pg_trim_to(), ..., pg.get_min_last_complete_ondisk,...)" which looks weird.